src/api/myanimelist.ts

```diff
@@ -0,0 +1,45 @@
+import * as request from "request";
+import rp from "request-promise";
+import { StringNullableChain } from "lodash";
+
+export enum MediumType {
+  anime = "anime",
+  manga = "manga"
+}
+
+export interface AnimeAndManga {
```

Comment:
Remember how we talked about not having the results in the interface? Instead, just return the results array directly?